packages/devtools/src/ganache/server.ts

```diff
@@ -137,29 +134,29 @@ export class GanacheServer {
     logger.info(`Starting ganache on port ${this.port} with network ID ${this.chainId}`);
     this.fundedPrivateKey = accounts[0].privateKey;
 
-    const oneMillion = ethers.utils.parseEther('1000000');
-
-    const args: string[] = [
-      [`--networkId ${this.chainId}`, `--port ${this.port}`],
-      accounts.map(a => `--account ${a.privateKey},${a.amount || oneMillion}`),
-      [`--gasLimit ${gasLimit}`, `--gasPrice ${gasPrice}`],
-      SHOW_VERBOSE_GANACHE_OUTPUT ? ['--verbose'] : []
-    ].reduce((a, b) => a.concat(b));
-
-    this.server = spawn('ganache-cli', args, {stdio: 'pipe', shell: true});
-    this.server.stdout.on('data', data => {
-      if (SHOW_VERBOSE_GANACHE_OUTPUT) {
-        extractLogsFromVerboseGanacheOutput(this.buffer, data.toString());
-      } else {
-        logger.info(data.toString());
+    const oneMillion = ethers.utils.parseEther('1000000').toHexString();
+
+    // ganache core exports a very permissive object[] type for accounts
+    // it should be {balance: HexString, secretKey: string}[]
+    const serverOptions: ganache.IServerOptions = {
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      network_id: this.chainId,
+      accounts: accounts.map(a => {
+        return {balance: oneMillion, secretKey: a.privateKey};
+      }),
+      gasLimit,
+      gasPrice: BigNumber.from(gasPrice).toHexString(),
+      logger: {
+        log: x => {
+          extractLogsFromVerboseGanacheOutput(x);
```

Comment:
I wasn't 100% sure of the value of a) piping through this function as opposed to b) piping straight to `logger.info`.

a produces logs like [this](https://gist.github.com/geoknee/765ff7b0f6c1935879dbe9bd9d7ee5c6). 
b produces logs like [this](https://gist.github.com/geoknee/1b902a552588ebc10036ae8ebe4093e8).

Reply:
(I went with 'a' so far)

Reply:
What about when verbose logging is off?

Reply:
See my other comment: I think the logging is always verbose and [no way to switch it of](https://www.npmjs.com/package/ganache-core#options)f. Should probably clean up some environment variables in light of that.

Reply:
I'll revise that to saying there is no way to switch verbosity, but whether this means verbose is ON or OFF or not applicable, I do not know.